modules/navier_stokes/include/postprocessors/NSCFLTimeStepSize.h

```diff
@@ -0,0 +1,55 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
```

Comment:
or object name?

Reply:
I cannot do either right now because of compatibility with THM. After this is merged, we can delete the file/object in THM and then we will rename this file.